packages/broadcast/component.js

```diff
@@ -1,46 +1,56 @@
-import { FabricWebComponent, windowExists } from '../utils';
+import { LitElement } from 'lit-element';
+import { windowExists } from '../utils';
+import { FabricToastContainer } from '../..';
```

Comment:
nit: might be nice to define an import for these so one can do `import { FabricToastContainer } from '#components'` - e.g. https://github.com/fabric-ds/vue/blob/main/components/forms/f-toggle.vue#L19

Reply:
oh yea, good point. will look into that